.github/workflows/helm-chart-ci.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google LLC
+# Copyright 2022 Google LLC
```

Comment:
**Rationale for this change:**
This is a nitpick.
I've updated the Apache license header years to 2022 for new files.